### PR TITLE
[ci skip] make xonsh script for switching bot status so I stop messing it up

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: switch.xsh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+if 'please.go' in @(ls):
+    mv please.go please.stop
+    git add please.stop
+    s = 'stop in the name of bot'
+else:
+    mv please.stop
+    git add please.go
+    s = 'let it go, let it gooooo'
+git commit -am @(f"[CI SKIP] {s}")
```
Comment (beckermr) [inline switch.xsh:9]: if you ci skip on starting the bot won't go.

File: switch.xsh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+if 'please.go' in @(ls):
+    mv please.go please.stop
+    git add please.stop
+    s = 'stop in the name of bot'
```
Comment (beckermr) [inline switch.xsh:4]:
```suggestion
    s = '[ci skip] stop in the name of bot'
```

File: switch.xsh (hunk 1 above)
Comment (beckermr) [inline switch.xsh:9]:
```suggestion
git commit -am @(f"{s}")
```

**Comment by CJ-Wright** [comment]
We should test this at some point, worst case we push some patches if needed.

File: switch.xsh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+if 'please.go' in @(ls):
```
Comment (beckermr) [inline switch.xsh:1]: Should we add some sort of usr bin env xonsh here and make the script executable?

**Review by beckermr — APPROVED** [review]

**Pull request merged**